sass/tokens/_parse.scss

```
@@ -256,7 +256,7 @@ $_stack: ();
       $alias-key: map.get($alias, 'alias');
       $lookup: lookup-alias($map, $alias-key);
 
-      // Replace #alias when lookup succeeds. Otherwise leave it in the string (eg #aaa)
+      // Replace #alias when lookup succeeds. Otherwise leave it in the string (e.g.#aaa)
```

```suggestion
      // Replace `#alias` when lookup succeeds. Otherwise leave it in the string (e.g. `#aaa`)
```

mostly just the missing space, but also added code fencing?